ee/clickhouse/queries/actor_base_query.py

```diff
@@ -54,34 +51,34 @@ def actor_query(self) -> Tuple[str, Dict]:
 
     @cached_property
```

Comment:
should these be cached? Aren't things too low level here to be caching?

Reply:
Not sure what you mean. That property doesn't need to be recalculated everytime. Also, it can be overridden and a child class can specify something more specific to the insight